indica/models/helike_spectroscopy.py

```diff
@@ -5,8 +5,8 @@
 from xarray import DataArray
 
 from indica.converters import LineOfSightTransform
+from indica.defaults.load_defaults import load_default_objects
```

Comment:
While we're refactoring, I would decouple the examples from the class files.

This solves some potential issues with circular imports.

I had started doing this some weeks ago (...never got back to it...) and had made an indica/examples/folder as in this picture:
![image](https://github.com/indica-mcf/Indica/assets/74655352/379e7d5a-f589-445d-94bd-33c566493de0)